packages/manager/apps/web/client/app/domain/dashboard/domain.service.js

```diff
@@ -38,6 +38,7 @@ angular.module('services').service(
       OvhHttp,
       Poll,
       Poller,
+      iceberg,
```

Comment:
Iceberg doesn't seem to be used here

Reply:
Oups sorry, You may have missed my last commit, I should have put it in review after, my bad ;)